pleskdistup/actions/systemd.py

```diff
@@ -176,6 +175,33 @@ def _revert_action(self) -> action.ActionResult:
         return action.ActionResult()
 
 
+# We need to handle the mailman service separately because it is not configured and is stopped by default
+# If we try to start an unconfigured service, it will fail, causing issues during the revert and finish stages
+class HandleMailmanService(action.ActiveAction):
```

Comment:
Seems that this action is not specific to Mailman. Can we just replace it with some generic `DisableServiceDuringUpgrade`?